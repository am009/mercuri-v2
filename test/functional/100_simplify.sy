int a = 1;
float u = 1.2;
int x;
int main() {
    int c = a + 2 - a;    
    putint(c);
    x = 10;
    putch(x);
    float d = a + 2 - a;
    putfloat(d);
    x = 10;
    putch(x);
    float e = u + 2 - u;
    putfloat(e);
    x = 10;
    putch(x);
    float k = u + 2.1 - u;
    putfloat(k);
    x = 10;
    putch(x);
    return 0;
}