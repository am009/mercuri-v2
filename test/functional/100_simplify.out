2
0x1p+1
0x1p+1
0x1.0cccccp+1
0